en/gps_compass/rtk_gps_gem1305.md

```diff
@@ -0,0 +1,144 @@
+- RTK accuracy: **2cm** +1PPM(H), 3cm+1PPM(V)
+
+## Interface
+
+- UART: 230400bps default
+- SMA connector for antenna
+- Output rate 5Hz default, up to 10Hz
+- Main power supply: 4.7~5.2V
+
+## Protocol
+
+- NMEA-0183 output
+- RTCMv3 input/output
+
+## Environment
+
+- Operating temp. -20~85°C
+
+## Dimension and Weight
+
+- 55x55x12mm
+- 50g (GEM1305) 26g (NANO with Helix)
+
+## Pinout
+
+The board is connected to the autopilot via UART interface.
+
+![GEM1305 connector](../../assets/hardware/gps/datagnss_gem1305/gem1305_connector.png)
+
+The 1.25mm pitch 6P connector (from left: PIN1 to PIN6):
+
+- 1: GND
+- 2: NC
+- 3: PPS
+- 4: Rx
+- 5: Tx
+- 6: 5V
+
+The board does not include a compass and so does not require connections for a compass part.
+
+## Hardware Setup
+
+RTK requires a base RTK module attached to the ground station, and a rover RTK module on the vehicle.
+The data from the base needs to be transmitted to the drone via telemetry radio and inputed into the RTK receiver on the rover.
+
+![RTK setup overview](../../assets/hardware/gps/datagnss_gem1305/setup_overview.png)
+
+The setup/connections for base and rover modules are shown below.
+
+### Base Setup (GCS)
+
+The base station connections are shown in the diagram below, along with the connection to a telemetry radio.
+
+![Base module setup](../../assets/hardware/gps/datagnss_gem1305/base_gnss_setup.png)
+
+Note that for the base we recommend the [NANO RTK Receiver](https://www.datagnss.com/collections/gnss-for-drone/products/multi-band-rtk-receiver-package) because it is easier to setup.
+
+<img src="../../assets/hardware/gps/datagnss_gem1305/nano_rtk_with_case.png" width="500px" alt="DATAGNSS NANO RTK Receiver">
+
+See to [How to setup Base station](https://wiki.datagnss.com/index.php/GEM1305-autopilot) for more details.
+
+### Rover Setup (PX4)
+
+The rover setup showing connections to a GPS port and the (required) connection to a telemetry radio are shown in the diagram below.
+
+![Rover module and pixhawk big picture connections](../../assets/hardware/gps/datagnss_gem1305/rover_gnss_setup.png)
+
+The diagram below shows how you might connect to the `GPS` port on the Pixawk 2.4.8 flight controller.
+The same connection approach might be used on a more recent flight controller.
+
+::: warning
+Pixawk 2.4.8 is a clone of a very old Pixhawk series design.
+Even though it is inexpensive it is not recommended: the board has such little memory that it can be difficult to find space for all the software components you need.
+It also uses non-Pixhawk standard connectors which are far less reliable, and works with many fewer components out of the box.
+:::
+
+![Rover module setup](../../assets/hardware/gps/datagnss_gem1305/pixhawk2_4_8_connections.png)
+
+GPS and RTK configuration on PX4 via _QGroundControl_ is plug and play (see [RTK GPS](../gps_compass/rtk_gps.md) for more information).
+
+## Package List
+
+- GEM1305 RTK Receiver
+- DG-6P-C01, GH-1.25mm-6P cable
```

Comment:
Something like

```suggestion
- Cables:
  - Pixhawk-standard Basic GPS (plugs directly into 6-pin `GPS 2` on Pixhawk-compatible controllers).
  - DG-6P-C01 - cable for connecting to older Pixhawk devices, such as Pixhawk 2.4.8
```

Reply:
Yes, but I'd still like images connecting to a recent flight controller. All the images show the old cable. When I see that I just think - nup, too much hassle.

Reply:
OK. Agree with you.
we will update the images ASAP.

Reply:
Thanks. New images good.